.github/workflows/main.yml

```
@@ -77,3 +77,15 @@ jobs:
           testTargetId: '35c8bfca-48d2-4eb2-8042-4ee50707a295'
           token: ${{ secrets.AUTOMAGICALLY_TOKEN }}
           environmentName: "test"
+
+      - name: Execute Automagically with overwriting variables
+        id: automagically-variable-overwrite
+        uses: ./
+        with:
+          url: 'https://storage.googleapis.com/mocktopus/index.html'
+          testTargetId: '35c8bfca-48d2-4eb2-8042-4ee50707a295'
+          token: ${{ secrets.AUTOMAGICALLY_TOKEN }}
+          environmentName: "test"
+          variablesToOverwrite: |-
+            VARIABLE_1:someValue
+            VARIABLE_2:someOtherValue
```

```suggestion
            VARIABLE_1: someValue
            VARIABLE_2: someOtherValue
```

can we support this for my sense of aesthetics? 😬